helm/tracker/values.yaml

```
@@ -75,6 +75,17 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+gateway:
+  enabled: true
```

```suggestion
  enabled: false
```

No all have a api gateway in kubernetes, we prefere disable by default.

---

Yep sure my bad ahaha forgot to turn it down